src/components/settings/Settings.scss

```diff
@@ -20,3 +20,49 @@
     background-color: #ffffff0a;
   }
 }
+
+.modal {
```

Comment:
I'd keep the modal in the center of the screen instead of beside the dropdown. Also since this confirmation modal is smaller in height than the default edit search button modal, you can add a subclass within .modal in SearchButton.scss. For example, &.-confirmation and set the height to be something else there. Then when you create a modal for the confirmation you use className "modal -confirmation"